python/paddle/jit/sot/opcode_translator/executor/function_graph.py

```diff
@@ -868,6 +868,25 @@ def gen_load_inputs(
                 self.pycode_gen.gen_load_const("int64")
                 self.pycode_gen.gen_call_function(3)
 
+    @staticmethod
+    def is_graph_output(
+        var,
+    ) -> TypeGuard[TensorVariable | SymbolicVariable]:
+        return isinstance(
+            var.tracker, (DummyTracker, SymbolicOperationTracker)
+        ) and isinstance(var, (TensorVariable, SymbolicVariable))
+
+    def collect_related_dummy_tensor(self, var):
```

Comment:
```suggestion
    @staticmethod
    def _collect_related_dummy_tensor(graph, var):
```

- `_find_tensor_outputs` 本就是「私有」，这两个也尽可能私有
- 这里 `collect_related_dummy_tensor` 从语义上不会直接 `graph.collect_related_dummy_tensor(var)`，而应该是 `FunctionGraph.collect_related_dummy_tensor(graph, var)`